### ISSUE TITLE
Explore Markdown parsing in posts and comment

### ISSUE THREAD

**Issue by docapotamus** [pjuu/pjuu#222]
Investigate allowing users to use Markdown in their posts and comments.

May need to be a subset and more restriction on comments than posts.

**Comment by dsohler** [comment]
Instead of allowing anything and then restrict it, I'd use awhitelist to allow only specific stuff. Since Github parses it too, I put it all into a code section. Wasn't it that newlines are possible now? imho that is mandatory.

~~~~
*text here*         for italic text

**text here**       for bold text

~~text here~~       for striked through text

[description](url)  for links (`<url>` alternatively for short notation
                    with automatic parsing to domain as link text)

`text here`         for code

1., 2., 3., ...     for having properly formatted lists (<ol>)

> text here         for quotes

# text here         for headings. I would't allow more levels of headings
                    than only `#` for `<h1>`

~~~~ and ~~~~       for code blocks with `<pre>` (block-level element)
                    while the mentioned notation of code via backticks
					uses `<code>` (inline element)
~~~~

For comments I wouldn't restrict anything except headings.

**Comment by docapotamus** [comment]
@dsohler I agree here all these features would be nice. Can't see any issues except for the line limits.

You can post as many new lines as you want but only 5 are shown in your feed. May find a way to limit this with JS instead of at the template.

**Comment by dsohler** [comment]
Maybe parse Markdown and minify the result to `<p>parsed Markdown here<p>` before counting the newlines? (And why is there a limitation of 5 lines?)

**Comment by docapotamus** [comment]
Just to stop people making really long posts and flooding your feed.

There are multiple ways to get around this though it will just be a case of trying which one is cleaner.
